post-scan-actions/aws-python-conformity-custom-check/template.yml

```diff
@@ -21,12 +21,18 @@ Parameters:
     Description: ARN of the File Storage SNS Topic to subscribe this function.
   ConformityRegion:
     Type: String
-    Description: Region where you conformity account is hosted. (us-west-2 for Cloud One customers)
-    Default: us-west-2
+    Description: Region where you conformity account is hosted. (us-1 for Cloud One customers)
+    Default: us-1
     AllowedValues:
-      - us-west-2
-      - ap-southeast-2
-      - eu-west-1
+      - us-1
+      - trend-us-1
```

Comment:
`trend-us-1` region is for internal use and is not supposed to be exposed to customers. Please remove it.

Reply:
Deleted.